packages/dd-trace/test/opentracing/span.spec.js

```diff
@@ -133,7 +133,25 @@ describe('Span', () => {
       span.setBaggageItem('foo', 'bar')
 
       expect(span.context()._baggageItems).to.have.property('foo', 'bar')
-      expect(parent._baggageItems).to.have.property('foo', 'bar')
```

Comment:
This assertion was not correct. The parent should contain a baggage item that was set on a descendant span.